documentation/Preprocessing.md

```diff
@@ -115,3 +120,100 @@ text.
 
 OpenMP-style directives that look like comments are not addressed by
 this scheme but are obvious extensions.
+
+Appendix
+========
+`N` in the table below means "not supported"; this doesn't
+mean a bug, it just means that a particular behavior was
+not observed.
+`E` signifies "error reported".
+
+The abbreviation `KWM` stands for "keyword macro" and `FLM` means
+"function-like macro".
+
+The first block of tests (`pp0*.F`) are all fixed-form source files;
+the second block (`pp1*.F90`) are free-form source files.
+
+```
+f18
+| pgfortran
+| | ifort
+| | | gfortran
+| | | | xlf
+| | | | | nagfor
+| | | | | |
+. . . . . .   pp001.F  keyword macros
```

Comment:
Are the tests going to be added somewhere? Otherwise the test names aren't meaningful.

Reply:
Right.  I'll slap copyright notices on them and push them out in another PR.